src/backend/cassandra/Schema.h

```diff
@@ -528,19 +528,21 @@ class Schema
                 SELECT hash, seq_idx 
                   FROM {}               
                  WHERE account = ?
-                   AND seq_idx < ?
+                   AND seq_idx <= ?
                  LIMIT ?
                 )",
                 qualifiedTableName(settingsProvider_.get(), "account_tx")));
         }();
 
+        // the smallest transaction idx is 0, we use uint to store the transaction index, so we shall use ">=" to
+        // include it(the transaction with idx 0) in the result
```

Comment:
This seems to be changing the output.. is this the same behaviour rippled has as far as output goes?

Reply:
This is to fix an old issue.
Imagine if we query all the transaction  > ( seq1 , 0) which old clio did, we actually missed all the transactions of seq1 with transaction idx 0 .